server/testdata/integration/backends/04_couper.hcl

```diff
@@ -0,0 +1,47 @@
+server {
+  endpoint "/" {
+    request "anything" {
+      url = "{{ .origin }}/headers"
+      backend = "httpbin"
+    }
+    request {
+      url = "{{ .origin }}/delay/1"
+      backend = "httpbin"
+    }
+    response {
+      body = "a"
+    }
+  }
+
+  endpoint "/named" {
+    request "anything" {
+      url = "{{ .origin }}/headers"
+      backend = "httpbin"
+    }
+    request "named" {
+      url = "{{ .origin }}/delay/1"
+      backend = "httpbin"
+    }
+    response {
+      body = "a"
+    }
+  }
+
+  endpoint "/default" {
+    request "anything" {
+      url = "{{ .origin }}/headers"
+      backend = "httpbin"
+    }
+    request {
+      url = "{{ .origin }}/delay/1"
+      backend = "httpbin"
+    }
+  }
+}
+
+definitions {
+  backend "httpbin" {
```

Comment:
* Change labels of `backend` and `request` blocks to make them look less httpbin-like.
* Change URLs as well (both backend requests are delayed, right?)

Reply:
Yes, all origin responses are delayed to get a 100% chance to trigger the "resource leak" otherwise it would be random which request will hit first.